Config/config.php

```diff
@@ -65,6 +65,7 @@
                     'mautic.core.model.notification',
                     'mautic.lead.model.field',
                     'mautic.plugin.model.integration_entity',
+                    'mautic.lead.model.dnc',
```

Comment:
There is no need for this additional as the AbstractIntegration class has only 15 constructor arguments. https://github.com/acquia/mc-cs/blob/epic-mautic3.3.3/app/bundles/PluginBundle/Integration/AbstractIntegration.php#L200-L216
Am I missing something?

Reply:
Yes, you should really check MAUT-6009 branch: https://github.com/acquia/mc-cs/blob/MAUT-6009/app/bundles/PluginBundle/Integration/AbstractIntegration.php#L237